astra/src/main/resources/admin-ui/index.html

```
@@ -0,0 +1,225 @@
+  </main>
+
+  <!-- Partition Assignment Modal -->
+  <div class="modal-overlay" id="modal-partition">
+    <div class="modal">
+      <div class="modal-header">
+        <h3>Update Partition Assignment</h3>
+        <button type="button" class="modal-close">&times;</button>
+      </div>
+      <form id="form-partition">
+        <label>Dataset Name
+          <input type="text" name="name" readonly>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)